### PR TITLE
Create desktop version for portfolio project

### PR DESCRIPTION
**Pull request by nicupop729**
##### In this branch, we adapt the web page to desktop version making the following settings:

- adapt the HTML for desktop toolbar; 
- in CSS using media queries:
     - change the display of toolbar;
     - change the display of projects in the section;
     - adapt the About-me section;
     - adapt the Contact form section;

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -404,9 +416,185 @@ form button:active {
...
+    border: 0.5px solid rgba(124, 124, 124, 0.356);
+    border-radius: 8px;
+  }
+
+  .project-img {
+    width: 100%;
+  }
+
+  .img {
+    height: auto;
```
Comment (Alexoid1) [inline style.css]: - [x] Kindly for every project image use the height value same as the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A2068)template.
Captura de pantalla_2021-09-22_23-56-29

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
Hi @nicupop729 ,

Impressive project!
You are doing a great job so far, and the project is looking good.
- No linter errors
- Follow Github-flow guidelines.
- Descriptive README.md file.
- Impressive project is almost the exact same design as the Figma template.

## Required Changes :recycle:
 - For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 1 above)
Comment (nicupop729) [reply style.css]: I had changed the height of every portfolio project and the height of the placeholder images as they are in the Figma template.

**Review by carlos-ssh — APPROVED** [review]
Hi @nicupop729, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**